bedrock/firefox/templates/firefox/new/onboarding/scene1.html

```diff
@@ -85,7 +85,21 @@
                 <li><a class="more" href="{{ url('firefox.family.index') }}">{{_('Discover all the ways to take Firefox on the go')}}</a></li>
               </ul>
             </div>
-            <button id="other-platforms-modal-link" class="hidden" type="button">{{_('Firefox for Other Platforms & Languages')}}</button>
+
+            <div id="other-platforms-languages-wrapper" class="hidden">
+            {% if l10n_has_tag('firefox_new_install_options') %}
+              <button id="other-platforms-modal-link" type="button">
```

Comment:
Totally agree. There's UX needed here now. Is a modal really best? What if it was just a section you could scroll/anchor link to? We'd have more room for context and design...